### PR TITLE
fix: Add support for CI secret

### PR DESCRIPTION
**Pull request by hundredacres**
## Description

Added CI secret and updated tests.

## Pipeline Reference

| Pipeline |
| -------- |
|  [avm.res.fabric.capacity](https://github.com/hundredacres/bicep-registry-modules/actions/workflows/avm.res.fabric.capacity.yml)        |

## Type of Change

- [X] Update to CI Environment or utilities (Non-module affecting changes)
- [ ] Azure Verified Module updates:
  - [ ] Bugfix containing backwards-compatible bug fixes, and I have NOT bumped the MAJOR or MINOR version in `version.json`:
    - [ ] Someone has opened a bug report issue, and I have included "Closes #{bug_report_issue_number}" in the PR description.
    - [ ] The bug was found by the module author, and no one has opened an issue to report it yet.
  - [ ] Feature update backwards compatible feature updates, and I have bumped the MINOR version in `version.json`.
  - [ ] Breaking changes and I have bumped the MAJOR version in `version.json`.
  - [ ] Update to documentation

## Checklist

- [X] I'm sure there are no other open Pull Requests for the same update/change
- [X] I have run `Set-AVMModule` locally to generate the supporting module files.
- [X] My corresponding pipelines / checks run clean and green without any errors or warnings

### CODE REVIEW

File: avm/res/fabric/capacity/tests/e2e/waf-aligned/main.test.bicep [hunk 1]
```diff
@@ -20,6 +20,10 @@ param serviceShort string = 'fcwaf'
 @description('Optional. A token to inject into the name of each resource. This value can be automatically injected by the CI.')
 param namePrefix string = '#_namePrefix_#'
 
+@description('Required. Email address used by resource. This value is tenant-specific and must be stored in the CI Key Vault in a secret named \'CI-MySecret\'.')
```
Comment (eriqua) [inline avm/res/fabric/capacity/tests/e2e/waf-aligned/main.test.bicep:23]: Please update description with the actual kv secrets name

File: avm/res/fabric/capacity/tests/e2e/defaults/main.test.bicep [hunk 2]
```diff
@@ -20,6 +20,11 @@ param serviceShort string = 'fcmin'
 @description('Optional. A token to inject into the name of each resource. This value can be automatically injected by the CI.')
 param namePrefix string = '#_namePrefix_#'
 
+@description('Required. Email address used by resource. This value is tenant-specific and must be stored in the CI Key Vault in a secret named \'CI-MySecret\'.')
```
Comment (eriqua) [inline avm/res/fabric/capacity/tests/e2e/defaults/main.test.bicep:23]: Please update description with the actual required kv secrets name

Comment (hundredacres) [reply avm/res/fabric/capacity/tests/e2e/defaults/main.test.bicep]: Fixed.

File: avm/res/fabric/capacity/tests/e2e/waf-aligned/main.test.bicep (hunk 1 above)
Comment (hundredacres) [reply avm/res/fabric/capacity/tests/e2e/waf-aligned/main.test.bicep]: Fixed.

**Review by eriqua — APPROVED** [review]
🆗

**Pull request merged**